include/BALL/MATHS/quaternion.h

```diff
@@ -200,7 +199,7 @@ namespace BALL
 		 * @param m the matrix to compute from
 		 * @return TMatrix4x4 the rotation matrix
 		 */
-		TMatrix4x4<T>& getRotationMatrix(TMatrix4x4<T>& m) const;
+		void getRotationMatrix(TMatrix4x4<T>& m) const;
```

Comment:
Do we need to break the interface here? I get that the fixed class does no longer provide non-const refs to the quaternion components via its getters. But the change to this function here does not seem to be necessary.

(Also, the `@return` descriptions of the functions with modified interfaces are now incorrect.)